src/app/public/modules/modal-provider/types/modal-provider-injection-token.ts

```diff
@@ -0,0 +1 @@
+export const SKY_MODAL_PROVIDER = 'sky_modal_provider';
```

Comment:
I believe best practice is to use `InjectionToken` here. For example: https://github.com/blackbaud/skyux-i18n/blob/master/src/app/public/modules/i18n/lib-resources-providers-token.ts

Reply:
Done. I had based this off of the `http` library but I'm always happy to learn a better way :)